_data/backup.yml

```diff
@@ -32,9 +32,11 @@ websites:
 
   - name: CrashPlan
     url: https://www.crashplan.com
-    twitter: crashplan
     img: crashplan.png
-    facebook: CrashPlan
+    tfa:
+      - software
+    doc: https://support.code42.com/Administrator/Small_Business/Get_started_with_CrashPlan_for_Small_Business/Two-factor_authentication_for_CrashPlan_for_Small_Business
+    exception: "Two-factor authentication affects access to the administration console and CrashPlan for Small Business mobile app. It does not affect access to the Code42 app on user computers."
```

Comment:
>It does not affect access to the Code42 app on user computers.
I can't make out if the Code42 app uses a different set of credentials or if it the same as for the admin console and CrashPlan for Small Business.
If it's the same then the site doesn't support 2FA according to our definition.